### PR TITLE
copr: Add vectorize space

### PR DESCRIPTION
**Pull request by tw666**
Signed-off-by: tw <t888y@qq.com>

PCP #5751

###  What have you changed?

Add vectorize space

###  What is the type of the changes?
- Improvement (a change which is an improvement to an existing feature)

###  How is the PR tested?
- Unit test
- Integration test

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?
No
###  Does this PR affect `tidb-ansible`?
No

### CODE REVIEW

**Comment by tw666** [comment]
@sre-bot /run-integration-copr-test copr-test=pr/90

File: components/tidb_query/src/rpn_expr/impl_string.rs [hunk 1]
```diff
@@ -238,6 +239,23 @@ pub fn locate_binary_3_args(
...
 
+#[rpn_fn]
+#[inline]
+pub fn space(len: &Option<Int>) -> Result<Option<Bytes>> {
+    Ok(match *len {
+        Some(len) => {
+            if len > i64::from(tidb_query_datatype::MAX_BLOB_WIDTH) {
+                None
+            } else if len <= 0 {
+                Some(b"".to_vec())
```
Comment (breeswish) [inline components/tidb_query/src/rpn_expr/impl_string.rs]: `Vec::new()`

File: components/tidb_query/src/rpn_expr/impl_string.rs [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 use std::str;
 use tidb_query_codegen::rpn_fn;
+use tidb_query_datatype;
```
Comment (breeswish) [inline components/tidb_query/src/rpn_expr/impl_string.rs]: You don't need this `use`.

**Comment by breeswish** [comment]
@shafreeck PTAL, thanks!

File: components/tidb_query/src/rpn_expr/impl_string.rs (hunk 2 above)
Comment (tw666) [reply components/tidb_query/src/rpn_expr/impl_string.rs]: updated

**Pull request merged**